pkg/controller/apischeme/apischeme_controller_test.go

```diff
@@ -5,185 +5,16 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/golang/mock/gomock"
-	configv1 "github.com/openshift/api/config/v1"
-	cloudingressv1alpha1 "github.com/openshift/cloud-ingress-operator/pkg/apis/cloudingress/v1alpha1"
-
-	awsclient "github.com/openshift/cloud-ingress-operator/pkg/awsclient/mock"
+	"github.com/openshift/cloud-ingress-operator/pkg/testutils"
 
+	configv1 "github.com/openshift/api/config/v1"
 	utils "github.com/openshift/cloud-ingress-operator/pkg/controller/utils"
-	machineapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/client-go/kubernetes/scheme"
-	"k8s.io/utils/pointer"
-	awsprovider "sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsproviderconfig/v1beta1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
 const masterMachineLabel string = "machine.openshift.io/cluster-api-machine-role"
```

Comment:
This appears to have been put into testutils (L23) -- why isn't it removed from here?

Reply:
Good catch. I'll remove once the Easter break is over.